app/src/pages/inside/common/expandedOptions/expandedOptions.scss

```
@@ -195,33 +191,24 @@
     pointer-events: none;
-    border-radius: 4px;
-
-    ~ .folders-tree {
-      opacity: 0.5;
-      pointer-events: none;
-    }
   }
 
   &__drop-placeholder-content {
     @include flex-center(column, center, center);
     gap: 16px;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)